bokeh/tests/test_resources.py

```diff
@@ -217,3 +218,23 @@ def test_argument_checks(self):
 
         for mode in ("inline", "cdn", "relative", "relative-dev", "absolute", "absolute-dev"):
             self.assertRaises(ValueError, resources.Resources, mode, root_url="foo")
+
+
+## Test external resources
+
+def test_external_js_and_css_resource_embedding():
+    """ This test method has to be at the end of the test modules because
+    subclassing a Model causes the CustomModel to be added as a Viewable and
+    messes up the Resources state for the other tests.
+    """
+
+    # External resources can be defined as a string or list of strings
+    class CustomModel(Model):
+        __javascript__ = "external_js_1"
+        __css__ = ["external_css_1", "external_css_2"]
+
+    r = resources.Resources()
+    print(r.js_files)
```

Comment:
`print` should be removed. Also I think it would be ideal to test all the possible cases, single and list for each `_javascript__` and `__css__` separately.